demos/provider-mock/src/api.ts

```diff
@@ -85,5 +88,153 @@ export function prosopoRouter(): Router {
 		},
 	);
 
+	router.get("/test", async (req, res) => {
+		console.log(req.headers);
+		console.log(req.rawHeaders);
+
+		// Extract headers
+		const headers = req.headers;
+
+		// Validate headers
+		if (
+			!headers["x-tls-clienthello"] ||
```

Comment:
should we just set the headers to '' rather than telling the caller they've not been set?